deploy

```diff
@@ -44,6 +45,13 @@ const { ECS, Credentials } = require('aws-sdk');
     const task = taskDefinitionDescription.taskDefinition;
     console.log(`Current task definition: ${task.taskDefinitionArn}`);
 
+    const containerEnvironmentVariables = (config.containerEnvPairs || [])
+      .map(keyValueString => {
+        const parts = keyValueString.split(/=(.+)/);
+        return { name: parts[0].trim(), value: parts[1].trim() };
+      })
+      .filter(envVariable => envVariable);
```

Comment:
How can this ever be false? :thinking:

Reply:
Ah, it used to be possible, I should add that back actually, removed it during testing.
I was checking if `parts.length < 2` and returning null, just to avoid a tool crash and pass the error on as missconfiguration basically.

Reply:
Removed this guy to let the tool crash.